### PR TITLE
Meiliu1313/imp 10 login page

### PR DESCRIPTION
**Pull request by me-liu**
### Description

[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- finished fixing styling so there isn't a space + everything lines up
- added colors into styling
- email is removed after input
- string type issues resolved

### Screenshots

[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"

image
image

Old
image
image
image

## How to review

[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

- there is commented information on the password/index file that is the functionality for the eye icon when it gets updated

## Next steps

[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- everything is done except for the eye button in the password + next arrow

### Online sources

[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @stephaniewong2

### CODE REVIEW

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import styles from './styles';
+import supabase from '../../../../supabase/createClient';
+
+export default function LoginScreen() {
+  const [email, setEmail] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+
+  async function emailFind(email: string) {
+    const { data, error } = await supabase
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: A bit of a nit but let's add this supabase query to the case queries file! Let me know if this doesn't makes sense I can explain more in person!

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      <View>
+        <TouchableOpacity
+          style={[styles.nextButton]}
+          onPress={() => emailFind(email)}
+        >
+          <Text>Next</Text>
+        </TouchableOpacity>
+      </View>
+
+      <Text style={styles.errorMessage}>
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx:87]: Let's add the error message above the next button like the Figma!

File: src/app/(Authentication)/Login/Email/styles.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    width: 310,
+    height: 50,
+    padding: 15,
+    opacity: 0.3,
+  },
+  errorMessage: {
+    color: '#920E1A',
+    fontSize: 16,
+    fontWeight: '400',
+    border: 30,
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/styles.ts]: `border` isn't a property of the `Text` component so we can remove this!

File: src/app/(Authentication)/Login/Email/styles.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    fontWeight: '800',
+    marginRight: 18,
+    marginLeft: 18,
+  },
+  input: {
+    marginTop: 100,
+    marginBottom: 60,
+    height: 40,
+    margin: 18,
+    borderWidth: 1,
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/styles.ts:81]: Instead of a box, we want a line so we can accomplish this by using `borderBottomWidth` instead of `borderWidth`

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+      <Text style={styles.instructionText}>
+        Please enter your email address.
+      </Text>
+      <TextInput
+        style={styles.input}
+        value={email}
+        onChangeText={setEmail}
+        placeholder="Email address"
+        keyboardType="email-address"
+        autoCapitalize="none"
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx:71]: `TextInput` has a `clearButtonMode` field and that can help us accomplish the "x" button!

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
+import { router, useLocalSearchParams } from 'expo-router';
+import React, { useState } from 'react';
+import { Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import supabase from '../../../../supabase/createClient';
+import { signInUser } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  // const [email, setEmail] = useState('');
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Remove unused state variable!

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  const [password, setPassword] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+  const { email } = useLocalSearchParams<{ email: string }>();
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: This is actually a little bit funky and still interprets the email variable as `string | undefined` but Ronnie found a workaround by using `const email = useLocalSearchParams() as unknown as string;`!

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+  async function signInFunc(email: string, pword: string) {
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Let's move queries to the auth queries folder!

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+        secureTextEntry
+      />
+      <TouchableOpacity
+        style={styles.nextButton}
+        onPress={() => signInFunc(email, password)}
+      >
+        <Text>Next</Text>
+      </TouchableOpacity>
+
+      <Text style={styles.errorMessage}>
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx:73]: Same comment as above but let's move the error message above the next button!

File: src/app/(Authentication)/Login/Password/styles.ts [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+    fontWeight: '800',
+    marginRight: 18,
+    marginLeft: 18,
+  },
+  input: {
+    marginTop: 130,
+    marginBottom: 60,
+    height: 40,
+    margin: 18,
+    borderWidth: 1,
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/styles.ts:70]: Same comment as before but let's use `bottomBorderWidth`

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+export default function LoginScreen() {
+  const [email, setEmail] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+
+  async function emailFind(email: string) {
+    const { data, error } = await supabase
+      .from('users')
+      .select()
+      .eq('email', email.toLowerCase());
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: Why do we lower case email?

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 12]
```diff
@@ -0,0 +1,61 @@
+import { Link, router } from 'expo-router';
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: Remove unused `Link` import

File: src/app/(Authentication)/Login/Password/index.tsx (hunk 8 above)
Comment (stephaniewong2) [reply src/app/(Authentication)/Login/Password/index.tsx]: We can do this by updating the sign in query to return a boolean so that we can use it in this file to determine whether or not to display the error message rather than writing two very similar queries

**Review by stephaniewong2 — CHANGES_REQUESTED** [review]
Looks really good!! Functionality is there!! I'll help you with styling and any questions whenever!

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+  const [email, setEmail] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+  const [displayEmail, setDisplayEmail] = useState(false);
+
+  function inputEmail(email: string) {
+    setDisplayEmail(true);
+    setEmail(email);
+  }
+
+  function removeEmail() {
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx:27]: Edge case: If a user is typing their email then clears it, the preview text "Email Address" is on both the line and above the text input. We only want this to be on one not both though let's follow up with Josh to see intended functionality!

File: src/app/(Authentication)/Login/Email/styles.ts [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+    fontSize: 13,
+    opacity: 0.35,
+  },
+  input: {
+    marginTop: 5,
+    height: 40,
+    borderBottomWidth: 2,
+    padding: 10,
+    width: 310,
+    borderRadius: 5,
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/styles.ts]: Since this is a straight line input we don't need any border radius!

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+        <Text style={styles.errorMessage}>
+          {' '}
+          {displayError
+            ? 'The email you entered is either incorrect or not registered with Impact Fund.'
+            : ' '}{' '}
+        </Text>
+      </View>
+
+      <View>
+        <TouchableOpacity style={[styles.nextButton]} onPress={() => emailFind}>
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: To call the function we either need to do `onPress={()=>emailFind()}` or `onPress={emailFind}`. Here we can do the latter for cleanliness since it does not take in any parameters

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+        placeholder="Email address"
+        keyboardType="email-address"
+        autoCapitalize="none"
+        clearButtonMode="while-editing"
+      />
+
+      <View style={styles.errorMessageBox}>
+        <Text style={styles.errorMessage}>
+          {' '}
+          {displayError
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: The text is cut off for me when the error message is displayed! Let's make sure styling accounts for this!

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 17]
```diff
@@ -0,0 +1,79 @@
+import { router, useLocalSearchParams } from 'expo-router';
+import React, { useState } from 'react';
+import { Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import { passwordExists, signInUser } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  const [password, setPassword] = useState('');
+  const [displayError, setDisplayError] = useState(false);
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx:10]: Same comment as above regarding email but the text is displayed twice when deleting password! Let's fix this after confirming intended functionality with Josh

File: src/app/(Authentication)/Login/Email/styles.ts [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+    borderColor: '#920E1A',
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/styles.ts]: `borderColor` should depend on whether the user is typing or not! It should be red if typing and black otherwise

File: src/app/(Authentication)/Login/Email/index.tsx (hunk 15 above)
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: For the "Next" button, let's add styling/logic so that if input is inputted, the button outline/text is black and otherwise it is grey

File: src/app/(Authentication)/Login/Password/styles.ts [hunk 19]
```diff
@@ -0,0 +1,76 @@
...
+    fontSize: 13,
+    opacity: 0.35,
+  },
+  input: {
+    marginTop: 5,
+    height: 40,
+    borderBottomWidth: 2,
+    padding: 10,
+    width: 310,
+    borderRadius: 5,
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/styles.ts]: Same comment as above but we don't need a border radius here!

File: src/app/(Authentication)/Login/Password/styles.ts [hunk 20]
```diff
@@ -0,0 +1,76 @@
...
+    borderColor: '#920E1A',
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/styles.ts]: Same comment as above, but this should depend on whether or not there is text in the input

File: src/app/(Authentication)/Login/Password/styles.ts [hunk 21]
```diff
@@ -0,0 +1,76 @@
...
+  },
+  nextText: {
+    marginRight: 200,
+    fontSize: 14,
+    opacity: 1,
+    fontWeight: '400',
+  },
+  nextButton: {
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/styles.ts]: Same comment as above but this should be black if there is input, otherwise grey like you have right now

File: src/supabase/queries/auth.ts [hunk 22]
```diff
@@ -57,3 +57,27 @@ export const signOutUser = async () => {
...
+  } else {
+    return true;
+  }
+};
+
+export const passwordExists = async (
+  email: string,
+  password: string,
+): Promise<boolean> => {
+  const { data } = await supabase.auth.signInWithPassword({
```
Comment (stephaniewong2) [inline src/supabase/queries/auth.ts:74]: Here `data` is never null and rather defined like this ```const data: {
    user: User;
    session: Session;
} | {
    user: null;
    session: null;
}``` so let's check for `data.user` instead!

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 23]
```diff
@@ -0,0 +1,79 @@
...
+  const email = useLocalSearchParams() as unknown as string;
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: This is actually a little funky and assigns `email` to `{"email": "email@email.com"}`. For now, I fixed this by doing `const { email } = useLocalSearchParams<{ email: string }>();
  const userEmail = email as string;` but I will do more research and get back to you if there is a better fix!

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 24]
```diff
@@ -0,0 +1,79 @@
...
+          {displayError
+            ? 'Oh no! The password you entered is incorrect, please try again.'
+            : ' '}{' '}
+        </Text>
+      </View>
+
+      <View>
+        <TouchableOpacity
+          style={[styles.nextButton]}
+          onPress={() => signInFunc(email, password)}
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Using what we have above now, let's pass in `userEmail` instead of `email``

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 25]
```diff
@@ -0,0 +1,79 @@
...
+import { passwordExists, signInUser } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  const [password, setPassword] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+  const email = useLocalSearchParams() as unknown as string;
+  const [displayPassword, setDisplayPassword] = useState(false);
+  // const [isPasswordVisible, setPasswordVisible] = useState(false);
+
+  function inputPassword(password: string) {
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Same little bug/edge case as above but if we type then clear all text, the display is both above and on the line.

**Review by stephaniewong2 — CHANGES_REQUESTED** [review]

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 26]
```diff
@@ -0,0 +1,99 @@
...
+    setIsFocused(true);
+    setDisplayEmail(true);
+  };
+
+  const offClick = () => {
+    setPlaceholder('Email');
+    setIsFocused(false);
+  };
+
+  function inputEmail(email: string) {
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: Let's remove this!

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 27]
```diff
@@ -0,0 +1,99 @@
...
+    setIsEmail(!isEmail);
+  }
+
+  async function emailFind() {
+    const isEmail = await emailExists(email);
+    if (!isEmail) {
+      setDisplayError(true);
+    } else {
+      setDisplayError(false);
+      if (typeof email === 'string') {
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: I don't think we need this check, we can just navigate to the next screen because we only accept text/string input

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 28]
```diff
@@ -0,0 +1,99 @@
...
+      <Text style={styles.instructionText}>
+        Please enter your email address.
+      </Text>
+      <Text style={styles.emailText}>
+        {displayEmail ? 'Email Address' : ' '}{' '}
+      </Text>
+      <TextInput
+        style={[styles.input, isFocused && styles.inputFocused]}
+        value={email}
+        onChangeText={inputEmail}
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: Let's call `setEmail` here instead

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 29]
```diff
@@ -0,0 +1,99 @@
...
+            ? 'The email you entered is either incorrect or not registered with Impact Fund.'
+            : ' '}{' '}
+        </Text>
+      </View>
+
+      <View>
+        <TouchableOpacity
+          style={[styles.nextButton]}
+          onPress={emailFind}
+          // disabled={!isEmail}
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: Remove if unused

File: src/app/(Authentication)/Login/Email/styles.ts
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/styles.ts:1]: Overall styling comments: let's make sure everything is aligned -- "Please enter...", "Email address", text input, error message, next button. Josh also wants the input line before inputting/empty input (in black) to be thinner but when text is inputted leave as is. Besides that, styling is good from my end. Josh will also look soon!

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 30]
```diff
@@ -0,0 +1,99 @@
...
+
+  async function emailFind() {
+    const isEmail = await emailExists(email);
+    if (!isEmail) {
+      setDisplayError(true);
+    } else {
+      setDisplayError(false);
+      if (typeof email === 'string') {
+        router.push({ pathname: 'Login/Password', params: { email } });
+      }
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: Let's set the `email` state variable to an empty string again after this!

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 31]
```diff
@@ -0,0 +1,108 @@
...
+import React, { useState } from 'react';
+import { Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import { passwordExists, signInUser } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  const [password, setPassword] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+  // const { email } = useLocalSearchParams<{ email: string }>();
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Let's do `const { email } = useLocalSearchParams() as unknown as { email: string };`

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 32]
```diff
@@ -0,0 +1,108 @@
...
+  // const userEmail = email as string;
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Remove commented out code!

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 33]
```diff
@@ -0,0 +1,108 @@
...
+    setPlaceholder('Email');
+    setIsFocused(false);
+  };
+
+  function inputPassword(password: string) {
+    if (password.trim() === '') {
+      setDisplayPassword(false);
+    }
+    setPassword(password);
+    /* if (password.trim() !== '') {
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Remove if not used

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 34]
```diff
@@ -0,0 +1,108 @@
...
+    */
+  }
+
+  function removePassword() {
+    if (password.trim() === '') {
+      setDisplayPassword(false);
+    }
+  }
+  /*
+  const togglePasswordVisibility = () => {
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Remove if unused -- we can bring this back when we handle the eye icon

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 35]
```diff
@@ -0,0 +1,108 @@
...
+    setPasswordVisible(!isPasswordVisible);
+  };
+  */
+
+  async function signInFunc() {
+    /*
+    if (typeof email !== 'string') {
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: remove

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 36]
```diff
@@ -0,0 +1,108 @@
...
+      setDisplayPassword(false);
+    }
+  }
+  /*
+  const togglePasswordVisibility = () => {
+    setPasswordVisible(!isPasswordVisible);
+  };
+  */
+
+  async function signInFunc() {
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Nit, but let's call this `signIn` instead of `signInFunc`

File: src/app/(Authentication)/Login/Password/styles.ts
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/styles.ts:1]: Same styling comments as login screen

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 37]
```diff
@@ -0,0 +1,108 @@
...
+import { passwordExists, signInUser } from '../../../../supabase/queries/auth';
+
+export default function LoginScreen() {
+  const [password, setPassword] = useState('');
+  const [displayError, setDisplayError] = useState(false);
+  // const { email } = useLocalSearchParams<{ email: string }>();
+  // const userEmail = email as string;
+  const email = useLocalSearchParams<{ email: string }>();
+  const [displayPassword, setDisplayPassword] = useState(false);
+  // const [isPassword, setIsPassword] = useState(false);
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Remove if unused

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 38]
```diff
@@ -0,0 +1,99 @@
...
+        keyboardType="email-address"
+        autoCapitalize="none"
+        clearButtonMode="while-editing"
+      />
+
+      <View style={styles.errorMessageBox}>
+        <Text style={styles.errorMessage}>
+          {' '}
+          {displayError
+            ? 'The email you entered is either incorrect or not registered with Impact Fund.'
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: There is a space before the first word in the error message -- " The email..." instead of "The email"

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 39]
```diff
@@ -0,0 +1,108 @@
...
+        placeholder={placeholder}
+        secureTextEntry
+        clearButtonMode="never"
+      />
+
+      <View style={styles.errorMessageBox}>
+        <Text style={styles.errorMessage}>
+          {' '}
+          {displayError
+            ? 'Oh no! The password you entered is incorrect, please try again.'
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: There is a space before the first word in the error message -- " Oh no..." instead of "Oh no"

**Review by stephaniewong2 — CHANGES_REQUESTED** [review]

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 40]
```diff
@@ -0,0 +1,90 @@
...
+        onBlur={offClick}
+        placeholder={placeholder}
+        keyboardType="email-address"
+        autoCapitalize="none"
+        clearButtonMode="while-editing"
+      />
+
+      <View style={styles.errorMessageBox}>
+        <Text style={styles.errorMessage}>
+          {' '}
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: I think this is causing the weird indent so we can remove this!

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 41]
```diff
@@ -0,0 +1,90 @@
...
+          {displayError
+            ? 'The email you entered is either incorrect or not \n registered with Impact Fund.'
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: Let's not add a new line in between because it is dependent on the screen itself

File: src/app/(Authentication)/Login/Email/index.tsx [hunk 42]
```diff
@@ -0,0 +1,90 @@
...
+        onEndEditing={removeEmail}
+        onFocus={onClick}
+        onBlur={offClick}
+        placeholder={placeholder}
+        keyboardType="email-address"
+        autoCapitalize="none"
+        clearButtonMode="while-editing"
+      />
+
+      <View style={styles.errorMessageBox}>
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/index.tsx]: I believe if you get rid of this View and update styling a bit to account for the button, the text will be properly aligned.

File: src/app/(Authentication)/Login/Password/index.tsx [hunk 43]
```diff
@@ -0,0 +1,89 @@
...
+        onFocus={onClick}
+        onBlur={offClick}
+        placeholder={placeholder}
+        secureTextEntry
+        clearButtonMode="never"
+      />
+
+      <View style={styles.errorMessageBox}>
+        <Text style={styles.errorMessage}>
+          {' '}
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/index.tsx]: Same comments as Email!

**Review by stephaniewong2 — CHANGES_REQUESTED** [review]

File: src/app/(Authentication)/Login/Email/styles.ts [hunk 44]
```diff
@@ -0,0 +1,92 @@
...
+    marginTop: 90,
+    marginRight: 225,
+    fontSize: 13,
+    opacity: 0.35,
+  },
+  input: {
+    marginTop: 5,
+    height: 40,
+    borderBottomWidth: 1.5,
+    padding: 10,
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/styles.ts:41]: Remove padding!

File: src/app/(Authentication)/Login/Email/styles.ts [hunk 45]
```diff
@@ -0,0 +1,92 @@
...
+    width: 315,
+    borderColor: colors.black,
+  },
+  inputFocused: {
+    marginTop: 5,
+    height: 40,
+    borderBottomWidth: 2,
+    padding: 10,
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/styles.ts]: Remove padding!

File: src/app/(Authentication)/Login/Password/styles.ts [hunk 46]
```diff
@@ -0,0 +1,94 @@
...
+    marginTop: 115,
+    marginRight: 240,
+    fontSize: 13,
+    opacity: 0.35,
+  },
+  input: {
+    marginTop: 5,
+    height: 40,
+    borderBottomWidth: 1.5,
+    padding: 10,
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/styles.ts]: Remove padding!

File: src/app/(Authentication)/Login/Password/styles.ts [hunk 47]
```diff
@@ -0,0 +1,94 @@
...
+    width: 310,
+    borderColor: colors.black,
+  },
+  inputFocused: {
+    marginTop: 5,
+    height: 40,
+    borderBottomWidth: 2,
+    padding: 10,
```
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Password/styles.ts]: Remove padding!

File: src/app/(Authentication)/Login/Email/styles.ts (hunk 44 above)
Comment (stephaniewong2) [inline src/app/(Authentication)/Login/Email/styles.ts]: Remove padding!

File: src/app/(Authentication)/Login/Password/styles.ts (hunk 21 above)
Comment (stephaniewong2) [reply src/app/(Authentication)/Login/Password/styles.ts]: Resolving because we can ignore for now
